linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -50,13 +52,20 @@ object Linker {
   case class LinkerConfig(
     namers: Option[Seq[NamerConfig]],
     routers: Seq[RouterConfig],
+    tracers: Option[Seq[TracerConfig]],
     admin: Option[Admin]
   ) {
     def mk: Linker = {
       val interpreter = namers.map(nameInterpreter)
 
-      val namersParam = Stack.Params.empty
-        .maybeWith(interpreter.map(DstBindingFactory.Namer(_)))
+      val tracerImpls = tracers.map(_.map(_.newTracer()))
+      val tracer: Tracer = tracerImpls match {
+        case Some(definedTracers) => BroadcastTracer(definedTracers)
+        case None => DefaultTracer
+      }
+
+      val params = Stack.Params.empty
+        .maybeWith(interpreter.map(DstBindingFactory.Namer(_))) + param.Tracer(tracer)
```

Comment:
oof yeah I forgot about wanting to trace namer requests. On the other hand I really want to pass the name interpreter into the tracer so I can use it to resolve the tracer's client's path :|